src/plugins/data/server/ui_settings.ts

```diff
@@ -34,6 +34,10 @@ import { UiSettingsParams } from 'opensearch-dashboards/server';
 // @ts-ignore untyped module
 import numeralLanguages from '@elastic/numeral/languages';
 import { DEFAULT_QUERY_LANGUAGE, UI_SETTINGS } from '../common';
+// cannot import from core/server due to src/core/server/saved_objects/opensearch_query.js which
+// export { opensearchKuery } from '../../../plugins/data/server';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { UiSettingScope } from '../../../core/server/ui_settings/types';
```

Comment:
will this impacts build?

Reply:
No. Actually, if importing from `core/server`, it only affects the unit tests in `create_or_upgrade_saved_config.test.ts`